core/config/project_settings.cpp

```diff
@@ -340,16 +335,35 @@ bool ProjectSettings::_set(const StringName &p_name, const Variant &p_value) {
 bool ProjectSettings::_get(const StringName &p_name, Variant &r_ret) const {
 	_THREAD_SAFE_METHOD_
 
+	if (!props.has(p_name)) {
+		WARN_PRINT("Property not found: " + String(p_name));
+		return false;
+	}
+	r_ret = props[p_name].variant;
+	return true;
+}
+
+Variant ProjectSettings::get_setting_with_override(const StringName &p_name) const {
+	_THREAD_SAFE_METHOD_
+
 	StringName name = p_name;
 	if (!disable_feature_overrides && feature_overrides.has(name)) {
```

Comment:
I believe nothing uses `disable_feature_overrides` anymore after this PR. So we could remove the boolean and the setter/getter.